backout https://github.com/di/calver/pull/21 for now
newer setuptools whines about the old syntax
older setuptools won't build with the new syntax

Index: pyproject.toml
--- pyproject.toml.orig
+++ pyproject.toml
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools>=77.0.1"]  # https://github.com/di/calver/pull/19
+requires = ["setuptools>=61.0"]  # https://github.com/di/calver/pull/19
 build-backend = "setuptools.build_meta"
 
 [project]
@@ -8,7 +8,7 @@ dynamic = ["version"]
 description = "Setuptools extension for CalVer package versions"
 readme = "README.md"
 requires-python = ">=3.9"
-license = "Apache-2.0"
+license = {text = "Apache-2.0"}
 authors = [
     { name = "Dustin Ingram", email = "di@python.org" }
 ]
